modules/docs/arrow-docs/docs/docs/datatypes/state/README.md

```diff
@@ -96,7 +91,7 @@ import arrow.typeclasses.*
 import arrow.instances.*
 
 fun stackOperations() = ForState<Stack>() extensions {
-  binding {
+  binding<StatePartialOf<Stack>, Option<String>> {
```

Comment:
The compiler trips here if the types are redundantly not specified. This is unrelated to Ank.